Iryna_Bondarenko/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Greet website</title>
+</head>
+<body>
+  <h2>1 route: /time</h2>
+  <h3> This route says the local time in Seattle</h3>
```

Comment:
I would make these `<h3>` tags into `<p>` tags - when you make everything "important", it just means nothing stands out as important anymore.